webhook/validation/validation.go

```diff
@@ -22,6 +26,10 @@ const (
 )
 
 const (
+	errorCSIRequired = `
+The DynaKube's specification tries to use the CSI driver while no CSI DaemonSet was deployed.
+Make sure that the deployed kubernetes/openshift.yaml contains the CSI DaemonSet.
```

Comment:
maybe: `The Dynakube's specification requires the CSI driver to work. Make sure you deployed the correct manifests.`